app/views/users/registrations/edit_profile.html.erb

```diff
@@ -20,7 +20,7 @@
 
   <div class="field">
     <%= f.label :phone %>
-    <%= f.text_field :phone, autofocus: true, :value => @user.phone %>
+    <%= f.text_field :phone, id: "phone", autofocus: true, :value => @user.phone %>
```

Comment:
Let's try to steer away from using `id`'s! Here are a few reasons why: http://oli.jp/2011/ids/